marketplacetests/marketplace/pages/base.py

```diff
@@ -36,11 +35,13 @@ def login(self, username, password):
         self.wait_for_login_success_notification()
 
     @property
-    def first_free_app_name(self):
+    def first_free_app(self):
         free_apps = self.search(':free').search_results
-        name = free_apps[0].name
-        self.tap_back()
-        return name
+        app = free_apps[0]
+        name = app.name
+        author = app.author
+        self.tap_home()
+        return {'name': name, 'author': author}
```

Comment:
I also considered returning multiple values, but most methods that call this only care about the name, and not the author, so the dictionary seemed better.